datamigration.py

```diff
@@ -261,12 +265,30 @@ def build_table_data(self, *args, **options):  # pylint: disable=too-complex, to
             if confirm('\nWould you like to provide a conditional to the data exporter?'):
                 self.table_data[table]['export_conditional'] = prompt('What would you like it to be? (include the WHERE)')
 
+        if options['media_from']:
+            self.media_from = options['media_from']
+
+        if options['media_to']:
+            self.media_to = options['media_to']
+
+        if confirm('Would you like to copy across media files from another project?'):
+            if not self.media_from:
+                self.media_from = prompt('Please enter the directory from which to copy media files:')
+            if not self.media_to:
+                self.media_to = prompt('Please enter the directory to copy media files to:')
+        else:
+            self.media_from = None
+            self.media_to = None
```

Comment:
I'm slightly concerned that they may be a path here where `media_from` and `media_to` aren't defined.  I'd suggest moving these two definitions above line 268 (the first `if options`).